client/src/pages/reports/Compact.js

```diff
@@ -0,0 +1,939 @@
+                          <ReadonlyCustomFields
+                            parentFieldName={`${Report.TASKS_ASSESSMENTS_PARENT_FIELD}.${task.uuid}`}
+                            fieldsConfig={taskInstantAssessmentConfig}
+                            values={report}
+                            vertical
+                            isCompact
+                          />
+                      )}
+                    </tbody>
+                  </table>
+                }
+              />
+            )
+          })}
+        </tbody>
+      </table>
+    )
+  }
+
+  function getAttendeesAndAssessments(attendees) {
+    // to keep track of different organization, if it is same consecutively, don't show for compactness
+    let prevDiffOrgName = ""
+    return (
+      <table>
+        <tbody>
+          {attendees.map(attendee => {
+            const attendeeInstantAssessmentConfig = attendee.getInstantAssessmentConfig()
+            const renderOrgName =
+              prevDiffOrgName !== attendee.position?.organization?.shortName
+            prevDiffOrgName = renderOrgName
+              ? attendee.position?.organization?.shortName
+              : prevDiffOrgName
+            return (
+              <CompactRow
+                key={attendee.uuid}
+                label={
+                  <>
+                    <LinkTo modelType="Person" model={attendee} />
+                    {renderOrgName && (
+                      <LinkTo
+                        modelType="Organization"
+                        model={
+                          attendee.position && attendee.position.organization
+                        }
+                        whenUnspecified=" 'NA'"
+                      />
+                    )}
+                  </>
+                }
+                content={
+                  optionalFields.assessments.active &&
+                  attendeeInstantAssessmentConfig && (
+                    <table>
+                      <tbody>
+                        <ReadonlyCustomFields
+                          parentFieldName={`${Report.ATTENDEES_ASSESSMENTS_PARENT_FIELD}.${attendee.uuid}`}
+                          fieldsConfig={attendeeInstantAssessmentConfig}
+                          values={report}
+                          vertical
+                          isCompact
+                        />
+                      </tbody>
+                    </table>
+                  )
+                }
+                style={`
+                  th {
+                    line-height: 1.4;
+                    width: max-content;
+                  }
+                  th label {
+                    margin-right: 4px;
+                  }
+                `}
+              />
+            )
+          })}
+        </tbody>
+      </table>
+    )
+  }
+  // first item is primary, grouped around that item
+  // people without organization at the end
+  function sortGroupByOrganization(attendees) {
```

Comment:
People without organization doesn't actually sorted to the end. In the following screenshot `OF-2 DVISOR, A` and `CIV HUNTMAN, Hunter` doesn't have a position.

![image](https://user-images.githubusercontent.com/62024178/100080092-76cacf00-2e56-11eb-9468-7f2a076fe09e.png)

I also think that sorting logic here can be simplified. Following patch file contains my suggestion.

[attendess-sort.patch.txt](https://github.com/NCI-Agency/anet/files/5589069/attendess-sort.patch.txt)